man/systemd.exec.xml

```diff
@@ -141,8 +141,10 @@
         <term><varname>Group=</varname></term>
 
         <listitem><para>Set the UNIX user or group that the processes are executed as, respectively. Takes a single
-        user or group name, or numeric ID as argument. If no group is set, the default group of the user is used. This
-        setting does not affect commands whose command line is prefixed with <literal>+</literal>.</para></listitem>
+        user or group name, or numeric ID as argument. If not user is explicitly set the user the invoking service
```

Comment:
"If the user is not explicitly set"? Also "inherited" is jargon-y. We should also say that this only works for the system instace. What about:

> For the system systemd instance, the default is "root". For the user instance, switching users is not supported, so the only setting that is valid is to specify the same user that the manager instance is running under.